includes/compatibility/class-wcpdf-compatibility-third-party-plugins.php

```diff
@@ -356,6 +359,42 @@ function restore_wgm_thumbnails( $document_type, $document ) {
 			add_filter( 'woocommerce_order_item_name', array( 'WGM_Product', 'add_thumbnail_to_order' ), 100, 3 );
 		}
 	}
+
+	/**
+	 * Adds invoice number filter to the search filters available in the admin order search.
+	 * 
+	 * @param array $options List of available filters.
+	 * 
+	 * @return array
+	 */
+	function hpos_admin_search_filters( array $options ): array {
+		if ( isset( get_option( 'wpo_wcpdf_documents_settings_invoice', array() )['invoice_number_search'] ) ) {
```

Comment:
There is a function you can use: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/3a63852e45c92fa2329251e0a5fb1d6bd0797c94/includes/class-wcpdf-admin.php#L402